### PR TITLE
install fix for #3

### PR DESCRIPTION
**Pull request by daften**

### CODE REVIEW

File: recipes/pear.rb [hunk 1]
```diff
@@ -7,26 +7,17 @@
...
-# PHPUnit PEAR channel
-php_pear_channel 'pear.phpunit.de' do
-  action :discover
-end
-
 # install/upgrade phpcs
 package = 'PHP_CodeSniffer'
 
 # upgrade when package is installed and latest version is required
+# TODO: This throws an error on first run, because pear is not installed yet
```
Comment (djoos) [inline recipes/pear.rb]: Not too sure about this @TODO-item, by this time PEAR should be installed already.
Would you be able to let me know what issue you had exactly on first run?

Thanks in advance!

Comment (daften) [inline recipes/pear.rb]: I think it's on compilation of the cookbooks. This is ruby code that is executed on compile. ruby code that should be executed on a chef run should be in a special resource (I think).
I get:
==> default: sh: pear: command not found
Tracing it further led me to that line

Comment (djoos) [inline recipes/pear.rb]: Hmmm, I can't seem to reproduce that. Which chef-client version are you running exactly?

Comment (daften) [inline recipes/pear.rb]: 12.0.3, it's on centos 7.0, i use a vagrantfile (was gonna make a PR for that, but I see you put the Vagrantfile in gitignore), want a pastebin for it? :)

Comment (djoos) [inline recipes/pear.rb]: Yeah, send me a gist or pastebin for it. I probably won't have time today, but will do my best over the next few days... Thanks in advance!

Comment (daften) [inline recipes/pear.rb]: Here ya go: https://gist.github.com/daften/a3641b089d8a39ea3ecc
No problem, thanks for looking into this ;)

**Pull request merged**